src/util/address.rs

```diff
@@ -85,11 +85,13 @@ pub enum Error {
     ExcessiveScriptSize,
     /// Script is not a p2pkh, p2sh or witness program.
     UnrecognizedScript,
+    /// Invalid address type.
+    InvalidAddressType(String),
 }
 
 impl fmt::Display for Error {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        match *self {
+        match &*self {
```

Comment:
We don't need this `&`, just use `(ref s)` below in `InvalidAddressType`.

Reply:
Makes sense! Thank you. This is done.